include/rangers-api/Application/Level/LevelInfo.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+namespace app::level {
+    class LevelInfo : public hh::game::GameService, public save::SaveManagerListener {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.